### PR TITLE
Add wakatime compact layout

### PR DESCRIPTION
**Pull request by WillianRod**
Adds Wakatime compact layout as sugested in the previous pull request and in this issue #504 

Here is the compact layout
a

Here is the normal layout
b

You can preview it [here](https://github-readme-stats-wa3h-avpqzc69j.vercel.app/api/wakatime?username=willianrod&layout=compact)

### CODE REVIEW

**Comment by WillianRod** [comment]
I would appreciated if you could label this PR as `hacktoberfest-accepted`, having in mind your repository is not opted in to the Hacktoberfest.

**Comment by WillianRod** [comment]
@anuraghazra Do you have any question?

**Comment by anuraghazra** [comment]
Hi @WillianRod thanks for the contribution, i'll review and merge the PR when i got some free time. :)

File: src/common/languageColors.json [hunk 1]
```diff
@@ -0,0 +1,430 @@
+{
+    "1C Enterprise": "#814CCC",
```
Comment (anuraghazra) [inline src/common/languageColors.json:2]: Woho nice! where did you get these colors? 🤔 from the GitHub Linguist repo?

File: src/cards/wakatime-card.js [hunk 2]
```diff
@@ -107,6 +143,71 @@ const renderWakatimeCard = (stats = {}, options = { hide: [] }) => {
...
+          noCodingActivityNode({
+            color: textColor,
+            text: i18n.t("wakatimecard.nocodingactivity"),
+          }),
+        ],
+      gap: lheight,
+      direction: "column",
+    }).join("");
+  }
+
```
Comment (anuraghazra) [inline src/cards/wakatime-card.js:210]: Hmmmmmm 🤔 really need to think how can we refactor this code into our reusable Card.js

File: src/common/languageColors.json (hunk 1 above)
Comment (WillianRod) [reply src/common/languageColors.json:2]: I got these colors from this repo https://github.com/ozh/github-colors/blob/master/colors.json

File: src/cards/wakatime-card.js (hunk 2 above)
Comment (WillianRod) [reply src/cards/wakatime-card.js:210]: How can we do this?

**Comment by WillianRod** [comment]
Meanwhile, can you add the label "hacktoberfest-accepted" to this PR?

**Comment by aios** [comment]
@WillianRod @anuraghazra Is the possible add "Exclusive wakatime card param" like `count` or `hideLessThan=5` **(mins)** for control count display languages?

**Comment by anuraghazra** [comment]
Theme preview (bot)  
      
      
title-color: <code>#faa627</code>
      
icon-color: <code>#faa627</code>
      
text-color: <code>#ffffff</code>
      
bg-color: <code>#36393f  }</code>
      
      
[](https://github-readme-stats.vercel.app/api?username=anuraghazra&title_color=faa627&icon_color=faa627&text_color=ffffff&bg_color=36393f  }&show_icons=true)

**Comment by anuraghazra** [comment]
Hey @WillianRod thanks for the PR and sorry for the late response. :)

**Comment by Muhammadyusuf96** [comment]
Shows error on compact layout: `Cannot read property 'length' of undefined`

**Pull request merged**